KsApi/models/graphql/adapters/GraphUserCreditCard+UserFragment.swift

```diff
@@ -4,20 +4,22 @@ extension GraphUserCreditCard {
   /**
    Returns a `GraphUserCreditCard` from a `GraphAPI.UserFragment`
    */
-  static func graphUserCreditCard(from userFragment: GraphAPI.UserFragment) -> GraphUserCreditCard? {
-    guard let storedCards = userFragment.storedCards,
-      let nodes = storedCards.nodes else { return nil }
+  static func graphUserCreditCard(from userFragment: GraphAPI.UserFragment) -> GraphUserCreditCard {
+    guard let storedCards = userFragment.storedCards?.fragments.userStoredCardsFragment.nodes else {
+      return GraphUserCreditCard(storedCards: [])
```

Comment:
Coverage is down a bit. This might be an easy place to write a test for to bump it up.